crates/build/src/build.rs

```diff
@@ -72,6 +72,8 @@ pub fn execute_build_program(
 
             std::fs::copy(&elf_path, &output_path)?;
         }
+    } else if args.elf_name.is_some() {
```

Comment:
cargo:warning=ELF name is set but --output-directory is not used. ELF will be written to `target/xyz`